src/plugins/data/public/query/filter_manager/lib/mappers/map_query_dsl.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { esFilters } from '../../../../../common';
+
+// Use mapQueryDsl mapper to avoid bloating meta with value and params for query DSL filters.
+export const mapQueryDsl = (filter: esFilters.Filter) => {
+  const metaProperty = /(^\$|meta)/;
```

Comment:
I feel like this should be `(^\$|^meta$)` (?)

but agreed that it should be consistent with https://github.com/thomasneirynck/kibana/blob/9769be3a289b4783a59988e9abf0cc2a5a1b6ba4/src/plugins/data/public/query/filter_manager/lib/mappers/map_default.ts#L24